owncloud/owncloud.py

```diff
@@ -39,41 +36,14 @@ def __init__(self, share_id, target_file, link, token):
 
     def __str__(self):
         return 'PublicShare(id=%i,path=%s,link=%s,token=%s)' % \
-               (self.share_id, self.target_file, self.link, self.token)
-
-
-class UserShare():
```

Comment:
Ok.
I really did the next.

with pi i downloaded the pyocclient
pip install pyocclient
and thext step I did

2to3  -w ......./lib/python3.4/site-packages/owncloud/__init__.py

after change the line
         headers['X-OC-MTIME'] = stat_result.st_mtime
by
         headers['X-OC-MTIME'] = bytes(str(stat_result.st_mtime),'utf-8')

and this is all

pyocclient works in python and owncloud 8 ^-^